### DISCUSSION TITLE
How to mask faces smaller than a certain area when  I use the node “inset special ”？

### DISCUSSION THREAD

**Discussion by 369386973** [nortikin/sverchok#4904] [Q&A] [answered]
![20230510211659](https://github.com/nortikin/sverchok/assets/101614185/a1d5b222-7449-4e44-b6be-812024fc1395)
I usually use list mask and logic function, but I failed in vector attraction.

**Comment by Durman** [comment]
By using List Mask (Out) node you change the size of lists. The Inset socket expects lengths of lists equal to number of polygons of input meshes. You can combine list with List Mask Join (In) node where polygons with small area would have on inset value and another polygons would have another insert value. Or you can use output of the Vector Attraction node directly and connect mask to the Ignore input to apply values only to small polygons.

![image](https://github.com/nortikin/sverchok/assets/28003269/2af95132-5503-42eb-bd92-0d357904ba43)

**Reply by 369386973** [reply]
![20230511141423](https://github.com/nortikin/sverchok/assets/101614185/e5e85f69-419b-47e4-b877-7a14148d83b5)
[case.zip](https://github.com/nortikin/sverchok/files/11448974/case.zip)
Sorry, I still don't quite understand...Can you explain with this small case（zip file）, for example, I want to interpolate the faces within the influence range of the vector attractor, but the faces less than 0.02 will not be interpolated.

**Reply by Durman** [reply, marked as answer]
I added a picture to the answer which may do the things more clear.

**Reply by 369386973** [reply]
Thanks！
